main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+	"github.com/ONSdigital/census-rm-pubsub-adapter/processor"
+	"log"
+	"os"
+	"os/signal"
+	"time"
+)
+
+func main() {
+
+	ctx := context.Background()
+
+	// Trap SIGINT to trigger a graceful shutdown.
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals, os.Interrupt)
+
+	a := processor.New(ctx, "amqp://guest:guest@localhost:6672/", "project")
```

Comment:
obvs it'll be good to make this configurable via OS env vars... but that can be done another time

Reply:
yep good point, thats added to our "things to do" trello card 👍